scripts/obtain/dotnet-install.ps1

```diff
@@ -366,6 +367,29 @@ function DownloadFile([Uri]$Uri, [string]$OutPath) {
     }
 }
 
+function Get-Runtime-Version-From-Dotnet-Package([System.IO.Compression.ZipArchive]$Zip) {
```

Comment:
Is there a reason this needs to be in this file? Can we not actually do it in the bootstrap script itself? Just find the highest version folder in under Microsoft.NETCore.App of the installed cli.